lambda/controller/validate-source/index.js

```diff
@@ -29,6 +31,19 @@ exports.handler = function(event, context, callback){
       }
     }
 
+    sourcedb.listTagsOfResource({ ResourceArn: sourceTable.Table.TableArn }, function(err, data) {
+      if(err){
+        return callback(new Error("Tags could not be listed for source table"));
+      } else {
+        for(const tag of data.Tags) {
```

Comment:
Please consistentify your spacing around () and {} etc. Where's prettier and eslint when you need it eh :)